app/assets/stylesheets/components/_card_list.css.sass

```diff
@@ -26,6 +26,12 @@
       &.ng-enter-active
         animation-name: bounceInLeft
 
+.CardList-card.is-closed
```

Comment:
Yes, but the CSS that styles the cards (which are the ones being affect) is given by `.CardList-card`, not by `.Discussion`.

That is the discussion itself, with the description and comments.

Should I keep it this way or update it?